Formula/lndir.rb

```diff
@@ -0,0 +1,22 @@
+class Lndir < Formula
+  desc "Create a shadow directory of symbolic links to another directory tree"
+  homepage "https://www.x.org"
```

Comment:
Is there no homepage that's more specific?

Reply:
Could it be the gitlab project page?

Reply:
Seems good to me.